### PR TITLE
Update workshop-n2women.md

### PR DESCRIPTION
**Pull request by michelenogueira**

### CODE REVIEW

File: workshop-n2women.md [hunk 1]
```diff
@@ -98,9 +98,9 @@ Presenting a poster is a great opportunity, especially for students, to obtain i
...
+Each submission should be formatted as an extended abstract, describing the research to be presented in the poster. The length of the extended abstract should be at most TWO pages (formatted into the US letter size of 8.5 x 11 inches with fonts no smaller than 10-point size), including all figures and references. The extended abstract must include the names, affiliations and email addresses of all authors and should be submitted as a single PDF file at <a href="https://n2women17.hotcrp.com/">https://n2women17.hotcrp.com/</a>.
 
-For more information, please contact the Workshop Chairs: n2women-workshop@acm.org
+For more information, please contact the Workshop Chairs: <a href="mailto:n2women-workshop@acm.org?Subject=N2Women%20Posters">n2women-workshop@acm.org</a>
```
Comment (cawka) [inline workshop-n2women.md:103]: you could write instead

    [n2women-workshop@acm.org](mailto:n2women-workshop@acm.org?Subject=N2Women%20Posters)

**Comment by michelenogueira** [comment]
Thanks.

File: workshop-n2women.md (hunk 1 above)
Comment (michelenogueira) [reply workshop-n2women.md:103]: Thanks. 

Alex, how could I control the size of the logos?
I would like to reduce the size of the google logo.
Also, I wil need to add another logo on the side of the google one.

Comment (cawka) [reply workshop-n2women.md:103]: You can change `"max-width: 50%"` property to something else. (e.g., `30%`

**Pull request merged**